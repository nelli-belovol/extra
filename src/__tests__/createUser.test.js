const { createUser } = require('../createUser');

describe('createUser', () => {
  it('should create a user with firstName and lastName only', () => {
    const user = createUser({
      firstName: 'John',
      lastName: 'Doe',
    });

    expect(user).toMatchSnapshot();
  });

  it('should create a user with firstName, lastName, and phone', () => {
    const user = createUser({
      firstName: 'Anton',
      lastName: 'Doe',
      phone: '123-456-7890',
    });

    expect(user).toMatchSnapshot();
  });

  it('should create a user with firstName, lastName, and email', () => {
    const user = createUser({
      firstName: 'John',
      lastName: 'Doen',
      email: 'dummy@email.com',
    });

    expect(user).toMatchSnapshot();
  });
});
