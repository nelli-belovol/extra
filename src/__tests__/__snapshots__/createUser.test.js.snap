// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createUser should create a user with firstName and lastName only 1`] = `
{
  "email": "",
  "firstName": "John",
  "lastName": "Doe",
  "password": "password",
  "phone": "",
  "type": "USER",
}
`;

exports[`createUser should create a user with firstName, lastName, and email 1`] = `
{
  "email": "dummy@email.com",
  "firstName": "John",
  "lastName": "Doen",
  "password": "password",
  "phone": "",
  "type": "USER",
}
`;

exports[`createUser should create a user with firstName, lastName, and phone 1`] = `
{
  "email": "",
  "firstName": "Anton",
  "lastName": "Doe",
  "password": "password",
  "phone": "123-456-7890",
  "type": "USER",
}
`;
